libexec/rtld-elf/rtld.c

```diff
@@ -1449,6 +1449,17 @@ digest_dynamic1(Obj_Entry *obj, int early, const Elf_Dyn **dyn_rpath,
 		if (dynp->d_un.d_val & DF_STATIC_TLS)
 		    obj->static_tls = true;
 	    break;
+
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(DT_CHERI___CAPRELOCS)
```

Comment:
We could probably drop the check for __CHERI_PURE_CAPABILITY__ if we want to allow global capabilities in hybrid mode. But for now it's probably better print the "Ignoring d_tag" message.

Reply:
Yeah, this was meant to be just an NFC refactor; functional changes for hybrid can come later when we have a toolchain that needs it.